src/tests/topp/MzTabExporter_1_output.mzTab

```diff
@@ -18898,53 +18901,53 @@ PEP	AITHLNNNFMFGQK	sp|P14866|HNRPL_HUMAN	1	null	null	null	0.0	0.0	0-UNIMOD:737,1
 PSM	AIFFNR	35	sp|Q99623|PHB2_HUMAN	1	uniprot_homo-sapiens_and_E-coli_June-2017_canonical-without-isoforms_reviewed_plus-contaminants_incl_bovinetrypsin_plus_shuffled	null	[, , MS-GF+, Release (v2017.08.23)]	4.129088340758449e-03	0-UNIMOD:737	2012.555219999999963	2	498.793487548828011	498.795048328071005	ms_run[1]:controllerType=0 controllerNumber=1 scan=11490	R	I	49	54	HCD	0	53.0	53.0	3.2672376e-09	0.072856635	3.2672376e-09	498.795043945313012	54	1	unique	49	0
 PSM	LFDHLK	36	sp|Q5F1R6|DJC21_HUMAN,dec_sp|P46100|ATRX_HUMAN	0	uniprot_homo-sapiens_and_E-coli_June-2017_canonical-without-isoforms_reviewed_plus-contaminants_incl_bovinetrypsin_plus_shuffled	null	[, , MS-GF+, Release (v2017.08.23)]	7.46975276170437e-03	0-UNIMOD:737,6-UNIMOD:737	2012.690039999999954	3	410.924072265625	410.925207618870957	ms_run[1]:controllerType=0 controllerNumber=1 scan=11491	K,A	A,Q	497,2471	502,2476	HCD	0	63.0	63.0	6.03261e-09	0.1345221	6.03261e-09	410.925201416016023	502	1	non-unique	497	0
 PSM	KAFSGYLGTDQSK	37	sp|P10768|ESTD_HUMAN	1	uniprot_homo-sapiens_and_E-coli_June-2017_canonical-without-isoforms_reviewed_plus-contaminants_incl_bovinetrypsin_plus_shuffled	null	[, , MS-GF+, Release (v2017.08.23)]	0.0	0-UNIMOD:737,1-UNIMOD:737,13-UNIMOD:737	2013.095880000000079	3	697.066284179687955	697.068076912904417	ms_run[1]:controllerType=0 controllerNumber=1 scan=11494	K	W	186	198	HCD	0	153.0	158.0	1.33755486e-17	3.2647832e-10	1.33755486e-17	697.068054199218977	198	1	unique	186	0
 PSM	SLDFYTR	38	sp|Q04760|LGUL_HUMAN	1	uniprot_homo-sapiens_and_E-coli_June-2017_canonical-without-isoforms_reviewed_plus-contaminants_incl_bovinetrypsin_plus_shuffled	null	[, , MS-GF+, Release (v2017.08.23)]	3.5799522673031e-04	0-UNIMOD:737	2013.939959999999928	2	565.804138183593977	565.805810923770991	ms_run[1]:controllerType=0 controllerNumber=1 scan=11499	K	V	45	51	HCD	0	75.0	80.0	4.866891e-10	0.011447002	4.866891e-10	565.805847167968977	51	1	unique	45	0
-PSM	AQLLVNPK	39	sp|Q13416|ORC2_HUMAN	1	uniprot_homo-sapiens_and_E-coli_June-2017_canonical-without-isoforms_reviewed_plus-contaminants_incl_bovinetrypsin_plus_shuffled	null	[, , MS-GF+, Release (v2017.08.23)]	2.40963855421687e-04	0-UNIMOD:737,8-UNIMOD:737	2014.885800000000018	2	670.934020996093977	670.936937418220964	ms_run[1]:controllerType=0 controllerNumber=1 scan=11506	R	K	40	47	HCD	0	74.0	77.0	6.760485e-11	1.610822e-03	6.760485e-11	670.936950683593977	47	1	unique	40	0
+PSM	AQLLVNPK	39	sp|Q13416|ORC2_HUMAN	1	uniprot_homo-sapiens_and_E-coli_June-2017_canonical-without-isoforms_reviewed_plus-contaminants_incl_bovinetrypsin_plus_shuffled	null	[, , MS-GF+, Release (v2017.08.23)]	2.40963855421687e-04	0-UNIMOD:737,8-UNIMOD:737	2014.885800000000018	2	670.934020996093977	670.936937418221078	ms_run[1]:controllerType=0 controllerNumber=1 scan=11506	R	K	40	47	HCD	0	74.0	77.0	6.760485e-11	1.610822e-03	6.760485e-11	670.936950683593977	47	1	unique	40	0
```

Comment:
can we limit the idigits in RT and m/z output  to e.g. 9 fractional digits?

Reply:
I don't think so. I think the float type gets lost due to conversion to MzTabDouble.

Reply:
We can only go through all occurrences where we write an RT and modify the output generator precision beforehand and change it back.

Reply:
ok let's keep for now